bin/.travis/prepare_unittest.sh

```diff
@@ -35,7 +35,7 @@ COMPOSER_UPDATE=""
 # solr package search API integration tests
 if [ "$TEST_CONFIG" = "phpunit-integration-legacy-solr.xml" ] ; then
     echo "> Require ezsystems/ezplatform-solr-search-engine:dev-master"
-    composer require --no-update ezsystems/ezplatform-solr-search-engine:dev-master
+    composer require --no-update ezsystems/ezplatform-solr-search-engine:~1.0.3
```

Comment:
That works, tested locally, thanks. Will update and keep when merging to master.